unit-test/helpers/mock-browser-api.js

```diff
@@ -1,8 +1,16 @@
 globalThis.browser = {
     storage: {
         local: {
-            set: () => {
+            set: () => {},
+            get: () => {
+                return {}
             }
         }
+    },
+    browserAction: {
+        setIcon: () => {}
+    },
+    runtime: {
+        sendMessage: () => {}
     }
 }
```

Comment:
this missing stuff was just producing a lot of annoying error messages when running tests